packages/numeric-input/src/presenters/SpinnerPresenter.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { cx, css } from "emotion";
+import { ThemeContext } from "@hig/theme-context";
+import { CaretUpMUI, CaretUpSUI, CaretDownMUI, CaretDownSUI } from "@hig/icons";
+import { createCustomClassNames } from "@hig/utils";
+import stylesheet from "./stylesheet";
+
+const variantTypes = ["line", "box"];
+
+export default class SpinnerPresenter extends Component {
+  static propTypes = {
+    disabled: PropTypes.bool,
+    onBlur: PropTypes.func,
+    onClick: PropTypes.func,
+    onFocus: PropTypes.func,
+    onChange: PropTypes.func,
+    onMouseDown: PropTypes.func,
+    onMouseEnter: PropTypes.func,
+    onMouseLeave: PropTypes.func,
+    onMouseUp: PropTypes.func,
+    variant: PropTypes.oneOf(variantTypes),
+    increment: PropTypes.func,
+    decrement: PropTypes.func,
+    stylesheet: PropTypes.func
+  };
+
+  static defaultProps = {
```

Comment:
Do you need this defaulted to `false`? You may be able to grab this from `...props` instead.